Podfile

```diff
@@ -11,13 +11,13 @@ workspace 'WordPress.xcworkspace'
 ##
 def wordpress_shared
     ## for production:
-    pod 'WordPressShared', '~> 1.8.9-beta.1'
+    # pod 'WordPressShared', '~> 1.8.8'
```

Comment:
You have an approved PR that updates to 1.8.10-beta-1 https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/225

Reply:
Done in a03947d233d1714dd0775fae4013c1c0e399db59